test/plugin/run.sh

```diff
@@ -113,8 +114,17 @@ do_cleanup() {
     [[ -d ${home}/workspace ]] && rm -rf ${home}/workspace
 }
 
+check_scenario_name_param() {
+    if test -z "$scenario_name"; then
+        echo "Missing value for the scenario argument"
+        exit 0
+    fi
+}
+
 start_stamp=`date +%s`
 parse_commandline "$@"
+check_scenario_name_param()
```

Comment:
have you run it? This statement is a syntax error. And it must be space below L129.
I suggest:
`test -z "$scenario_name" && exitWithMessage "Missing value for the scenario argument"`

BTW, remove unnecessary empty lines, Please.

Reply:
o, () must be delete. copy error

Reply:
o, copy error. () must be delete

Reply:
@dmsolr please review